src/components/TaskCard/AddTaskCard.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import { v1 as uuidv1 } from "uuid";
+import { sanitizeText } from "utils/helpers/sanitizeText.js";
+import PropTypes from "prop-types";
+import { addTask } from "store/actions/";
+import style from "components/TaskCard/index.module.scss";
+import { ENTER } from "utils/constant";
+const AddTaskCard = ({ showCreateCard, setShowCreateCard }) => {
```

Comment:
callback props should start with a prefix `on`

```suggestion
const AddTaskCard = ({ showCreateCard, onCreateCard }) => {
```